pkg/apis/extensions/v1beta1/types.go

```diff
@@ -968,27 +968,27 @@ type HostPortRange struct {
 	Max int32 `json:"max"`
 }
 
-// SELinux Context Strategy Options defines the strategy type and any options used to create the strategy.
-type SELinuxContextStrategyOptions struct {
+// SELinux Strategy Options defines the strategy type and any options used to create the strategy.
+type SELinuxStrategyOptions struct {
 	// type is the strategy that will dictate the allowable labels that may be set.
-	Type SELinuxContextStrategy `json:"type"`
-	// seLinuxOptions required to run as; required for MustRunAs
+	Type SELinuxStrategy `json:"type"`
+	// Options required to run as; required for MustRunAs
 	// More info: http://releases.k8s.io/HEAD/docs/design/security_context.md#security-context
-	SELinuxOptions *v1.SELinuxOptions `json:"seLinuxOptions,omitempty"`
```

Comment:
I knew that was why you suggested the change.  I just didn't know whether you were aware of the inconsistency, and whether you care about consistency or stuttering more.